<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="operator">
       <attribute name="title">
        <string>Оператор</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableView" name="operatorTable"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QComboBox" name="operatorDeviceTypeFilter">
              <item>
               <property name="text">
                <string>Все</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Арена</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ураган</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="operatorDateAfter"/>
            </item>
            <item>
             <widget class="QDateEdit" name="operatorDateBefore"/>
            </item>
            <item>
             <widget class="QComboBox" name="operatorState">
              <item>
               <property name="text">
                <string>Все</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Снятые</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>В работе</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Готовые</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="operatorRefreshButton">
              <property name="text">
               <string>Обновить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="operatorAddIncidentButton">
              <property name="text">
               <string>Добавить!!!</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="arena">
       <attribute name="title">
        <string>Ремонт арен</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTableView" name="arenaTable"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QDateEdit" name="arenaDateAfter"/>
            </item>
            <item>
             <widget class="QDateEdit" name="arenaDateBefore"/>
            </item>
            <item>
             <widget class="QComboBox" name="arenaState"/>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="arenaRefresh">
              <property name="text">
               <string>Обновить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="uragan">
       <attribute name="title">
        <string>Ремонт Ураган-ЮГ</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QTableView" name="uraganTable"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QDateEdit" name="uraganDateAfter"/>
            </item>
            <item>
             <widget class="QDateEdit" name="uraganDateBefore"/>
            </item>
            <item>
             <widget class="QComboBox" name="uraganState"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="uraganRefresh">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="usermanagement">
       <attribute name="title">
        <string>Управление пользователями</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="more">
       <attribute name="title">
        <string>Дополнительно</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="connectToDB"/>
    <addaction name="settings"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="connectToDB">
   <property name="text">
    <string>Соединиться с БД</string>
   </property>
  </action>
  <action name="settings">
   <property name="text">
    <string>Настройки программы</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
